patches/minecraft/net/minecraft/command/arguments/ArgumentTypes.java.patch

```diff
@@ -0,0 +1,13 @@
+--- a/net/minecraft/command/arguments/ArgumentTypes.java
++++ b/net/minecraft/command/arguments/ArgumentTypes.java
+@@ -220,4 +220,10 @@
+          this.field_197481_c = p_i48088_3_;
+       }
+    }
++
```

Comment:
Could you remove the empty lines here? 
Or is this policy to leave them here?
Also is a ForgeMarker not required here?

Reply:
I don't think we have the Forge marker in very many places anymore, but I don't know. I'll remove the empty line.